source/api/discovery/0.1/index.md

```diff
@@ -0,0 +1,609 @@
+
+As the number of Activities is likely too many to usefully be represented in a single Page, they are collected together into a Collection as the initial entry point. The Collection references the URIs of the first and last pages.
+
+```
+{
+  "id": "https://example.org/activity/all-changes",
+  "type": "OrderedCollection",
+  "totalItems": 21456,
+  "first": {
+  	"id": "https://example.org/activity/page-0",
+  	"type": "OrderedCollectionPage"
+  },
+  "last": {
+  	"id": "https://example.org/activity/page-214",
+  	"type": "OrderedCollectionPage"
+  }
+}
+```
+
+
+## 3. Activity Streams Details
+
+The W3C [Activity Streams][org-w3c-activitystreams] specification defines a "model for representing potential and completed activities", and is compatible with the [design patterns][annex-patterns] established for IIIF APIs. It is defined in terms of [JSON-LD][org-w3c-json-ld], and can be seamlessly integrated with the existing IIIF APIs. The model can be used to represent activities of creating, updating and deleting (or otherwise de-publishing) IIIF resources, carried out by content publishers.
+
+This section is a summary of the properties and types used by this specification, and defined by Activity Streams.  This is intended to ease implementation efforts by collecting the relevant information together.
+
+Properties, beyond those described in this specification, that the consuming application does not have code to process _MUST_ be ignored.  Other properties defined by Activity Streams _MAY_ be used, such as `origin` or `instrument`, but there are no current use cases that would warrant their inclusion in this specification.
+
+### 3.1. OrderedCollection
+
+The top-most resource for managing the lists of Activities is an Ordered Collection, broken up into Ordered Collection Pages. This is the same pattern that the Web Annotation model uses for Annotation Collections and Annotation Pages. The Collection does not directly contain any of the Activities, instead it refers to the `first` and `last` pages of the list.  
+
+The overall ordering of the Collection is from the oldest Activity as the first entry in the first page, to the most recent as the last entry in the last page. Consuming applications _SHOULD_ therefore start at the end and walk backwards through the list, and stop when they reach a timestamp before the time they last processed the list.
+
+Content providers _MUST_ publish an Ordered Collection at the HTTP(S) URI listed in the `id` property of the Collection.
+
+##### id
+
+The identifier of the Ordered Collection.
+
+Ordered Collections _MUST_ have an `id` property. The value _MUST_ be a string and it _MUST_ be an HTTP(S) URI. The JSON representation of the Ordered Collection _MUST_ be available at the URI.
+
+```
+{ "id": "https://example.org/activity/all-changes" }
+```
+
+##### type
+
+The class of the Ordered Collection.
+
+Ordered Collections _MUST_ have a `type` property.  The value _MUST_ be "OrderedCollection".
```

Comment:
are we going to adopt the new code-font styling for literals like `OrderedCollection`?

Reply:
Yes, will do this through out. :+1: